docs/src/_includes/components/language-switcher.html

```diff
@@ -14,9 +14,14 @@
         <select name="language selector" id="language-select" aria-describedby="language-infobox"
             class="c-custom-select switcher__select">
             {% for key, other_site in sites %}
-            <option value="{{ other_site.language.code }}" data-url="https://{{ other_site.hostname }}" {% if
-                site.language.code==other_site.language.code %} selected {% endif %}>
+            <option
+                value="{{ other_site.language.code }}"
+                data-url="https://{{ other_site.hostname }}/docs/latest{{ page.url | prettyURL }}"
+                {% if site.language.code==other_site.language.code %}selected{% endif %}
+            >
                 {{ other_site.language.flag }} {{ other_site.language.name }}
+                {% if site.language.code!=other_site.language.code %}({{ other_site.footer.language_switcher.latest }})
+                {% elif GIT_BRANCH=="latest" %}({{ other_site.footer.language_switcher.latest }}){% endif %}
```

Comment:
Can this be simplified with `or`?

```suggestion
                {% if site.language.code!=other_site.language.code or GIT_BRANCH=="latest" %}({{ other_site.footer.language_switcher.latest }}){% endif %}
```

Reply:
No, because `elif` is executed only if the first condition is `false`. If `or` is used, under `BRANCH=latest` all languages will be `latest`